### PR TITLE
Update trace-analytics.md

### PR DESCRIPTION
**Pull request by hdhalter**
Changed a file name, so fixing a broken link.

### Description
_Describe what this change achieves._

### Issues Resolved
Fixes jekyll error - https://github.com/opensearch-project/documentation-website/actions/runs/8382230525/job/22955582012

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The [OpenTelemetry source]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/c
 
 There are three processors for the trace analytics feature:
 
-* *otel_traces_raw* - The *otel_traces_raw* processor receives a collection of [span](https://github.com/opensearch-project/data-prepper/blob/fa65e9efb3f8d6a404a1ab1875f21ce85e5c5a6d/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records from [*otel-trace-source*]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/otel-trace/), and performs stateful processing, extraction, and completion of trace-group-related fields.
+* *otel_traces_raw* - The *otel_traces_raw* processor receives a collection of [span](https://github.com/opensearch-project/data-prepper/blob/fa65e9efb3f8d6a404a1ab1875f21ce85e5c5a6d/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records from [*otel-trace-source*]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/otel-trace-source/), and performs stateful processing, extraction, and completion of trace-group-related fields.
```
Comment (natebower) [inline _data-prepper/common-use-cases/trace-analytics.md:41]:
```suggestion
* *otel_traces_raw* - The *otel_traces_raw* processor receives a collection of [span](https://github.com/opensearch-project/data-prepper/blob/fa65e9efb3f8d6a404a1ab1875f21ce85e5c5a6d/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records from [*otel-trace-source*]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/otel-trace-source/) and performs stateful processing, extraction, and completion of trace-group-related fields.
```

Comment (natebower) [inline _data-prepper/common-use-cases/trace-analytics.md:41]: Please change the hyphen separating the term and the description to an en dash. Also, are the italics used in this list necessary?

File: _data-prepper/common-use-cases/trace-analytics.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ The [OpenTelemetry source]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/c
...
 * *otel_traces_group* - The *otel_traces_group* processor fills in the missing trace-group-related fields in the collection of [span](https://github.com/opensearch-project/data-prepper/blob/298e7931aa3b26130048ac3bde260e066857df54/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/trace/Span.java) records by looking up the OpenSearch backend.
```
Comment (natebower) [inline _data-prepper/common-use-cases/trace-analytics.md:42]: Please change the hyphen separating the term and the description to an en dash.

**Review by natebower — CHANGES_REQUESTED** [review]
@hdhalter Please see my comments and changes and let me know if you have any questions. Thanks!

File: _data-prepper/common-use-cases/trace-analytics.md (hunk 2 above)
Comment (hdhalter) [reply _data-prepper/common-use-cases/trace-analytics.md:42]: Good eye. :)

**Review by natebower — APPROVED** [review]
I think they may now all be hyphens, but otherwise, LGTM.

File: _data-prepper/common-use-cases/trace-analytics.md (hunk 1 above)
Comment (hdhalter) [reply _data-prepper/common-use-cases/trace-analytics.md]: > Please change the hyphen separating the term and the description to an en dash. Also, are the italics used in this list necessary?

Probably not

**Comment by hdhalter** [comment]
> I think they may now all be hyphens, but otherwise, LGTM.

Fixed!

**Pull request merged**